packages/forma-36-react-components/src/components/Button/README.md

```diff
@@ -0,0 +1,117 @@
+# Button
+
+Buttons communicate the action that will occur when the user clicks them. They communicate calls to action to the user and allow users to interact with pages in a variety of ways. They contain a text label to describe the action, and an icon if appropriate.
+
+## Button types
+
+- **Primary** - Used for the most important actions in any scenario. Don’t use more than one primary button in a section or screen to avoid overwhelming users.
+- **Muted** - Used for a secondary actions, the most commonly used button type.
+- **Positive** - For use when the action has a positive connotation such as creating or publishing a new entity.
+- **Negative** - For destructive actions - when something can't be undone. For example, deleting entities.
+- **Naked** - Link button
+
+
+![button overview](images/img.png)
```

Comment:
Is it a good idea to add screeshots to this docs?

Reply:
I think we should use code blocks so we can render them with MDX - removes the overhead of having to update images for changes in appearance

Reply:
@m10l yeah, you’re right. I removed them.